python/demo/demo_tnt-elements.py

```diff
@@ -32,6 +33,8 @@
 
 from mpi4py import MPI
 
+matplotlib.use('agg')
```

Comment:
@jorgensd could you check that this doesn't affect the Python docs? The docs run this demo to created plots.

Reply:
Runs fine locally:

![image](https://user-images.githubusercontent.com/8554300/228183624-d96a9a45-20a2-44a4-8699-6d22a989c8ff.png)

![image](https://user-images.githubusercontent.com/8554300/228183500-b8b483d3-b26a-4839-8f52-cc15417dbcdd.png)